tools/c7n_org/c7n_org/cli.py

```diff
@@ -458,6 +458,599 @@ def report(config, output, use, output_dir, accounts,
+        # Check if deprecations should fail in strict mode
+        if check_mode == deprecated.STRICT and accounts_with_warnings:
+            log.error("Deprecations found with --check-deprecations=strict")
+            return False
+        return True
+    else:
+        log.error(f"Validation failed for {len(accounts_with_errors)} account(s): "
+                  f"{', '.join(accounts_with_errors)}")
+        return False
+
+
+@cli.command()
+@click.option('-c', '--config', required=True, help="Accounts config file")
+@click.option('-u', '--use', required=True, help="Policy config file(s)")
+@click.option('-p', '--policy', multiple=True, help="Policy name filter")
+@click.option('-l', '--policytags', 'policy_tags',
+              multiple=True, default=None, help="Policy tag filter")
+@click.option('--resource', default=None, help="Resource type filter")
+@click.option('-a', '--accounts', multiple=True, default=None,
+              help="Account name or id filter")
+@click.option('--tags', multiple=True, default=None,
+              help="Account tag filter")
+@click.option('--not-accounts', multiple=True, default=None,
+              help="Exclude accounts")
+@click.option('--per-account', default=False, is_flag=True,
+              help="Validate per account with variable expansion")
+@click.option('--check-deprecations',
+              type=click.Choice(['skip', 'warn', 'strict']),
+              default='warn',
+              help="Check for deprecated features")
+@click.option('--debug', default=False, is_flag=True, help="Enable debug logging")
+@click.option('-v', '--verbose', default=False, help="Verbose output", is_flag=True)
+def validate(config, use, policy, policy_tags, resource,
+             accounts, tags, not_accounts, per_account,
+             check_deprecations, debug, verbose):
+    """validate policy files for c7n-org execution."""
+    # Setup logging
+    level = logging.DEBUG if verbose else logging.INFO
+    logging.basicConfig(
+        level=level,
+        format="%(asctime)s: %(name)s:%(levelname)s %(message)s"
+    )
+    # Suppress noisy third-party loggers
+    logging.getLogger('botocore').setLevel(logging.ERROR)
+    logging.getLogger('urllib3').setLevel(logging.ERROR)
+    logging.getLogger('oci').setLevel(logging.ERROR)
+    logging.getLogger('oci.circuit_breaker').setLevel(logging.ERROR)
+
+    log.info("Starting policy validation")
+    if verbose:
+        log.debug(f"Config file: {config}")
+        log.debug(f"Policy file: {use}")
+
+    # Import validation utilities from c7n.commands
+    from c7n.commands import DuplicateKeyCheckLoader
+    from c7n import deprecated
+    from c7n.resources import load_available
+
+    # Load available resources
+    load_available()
+
+    # Load and validate account config
+    log.debug("Loading and validating account config")
+    try:
+        with open(config, 'rb') as fh:
+            accounts_config = yaml.safe_load(fh.read())
```

Comment:
Thank you for using the all-important `safe_load` here! :tada: No YAML bomb imports/exploits! :tada: